exampleSite/data/content.yaml

```diff
@@ -1,7 +1,7 @@
 BasicInfo:
   FirstName: Inês
   LastName: Almeida
-  Photo: img/avatar.jpg
+  Photo: /img/avatar.jpg
```

Comment:
With this change, the avatar image as it is in the example site doesn't seem to show up. Any purpose for this change?

Reply:
sorry I forgot to answer!
The problem is that without this change the image doesn't show up on any translated page because it wants to get the image from e.g. http://example.com/de/img/avatar.png instead of http://example.com/img/avatar.png.

I don't know why it breaks for your test site tho as it just becomes an absolute url from a relative url which should still work :/

Reply:
That's very odd, I can't reproduce it... I've tried locally and remotely. No one has ever mentioned this, so I'd be more comfortable reverting this change. I'll add it back if needed.

Reply:
Thank's again for contributing, these are very nice changes!